packages/dx-grid-core/src/utils/table.test.js

```diff
@@ -38,15 +37,4 @@ describe('table utils', () => {
       expect(getTableTargetColumnIndex(columnGeometries, 1, 260)).toEqual(3);
     });
   });
-
-  describe('#getAnimations', () => {
-    it('should not return animations if columns are the same', () => {
```

Comment:
Why you delete this test? The getAnimation function has not cover with a test.

Reply:
This test is no longer needed. There are no other tests for this function.